### PR TITLE
[VA-3232] XDM support for Player State Tracking

### PR DESCRIPTION
**Pull request by midomir**
Issue #871

### CODE REVIEW

File: schemas/context/player-state.schema.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "player-state": {
+      "properties": {
+        "xdm:playerStateSet": {
+          "title": "Player state set",
+          "type": "boolean",
+          "description": "Whether or not the player state is set on that state."
+        },
+        "xdm:playerStateCount": {
+          "title": "Player state count",
+          "type": "number",
```
Comment (lrosenthol) [inline schemas/context/player-state.schema.json:25]: Should that be an integer?  What about a min/max?

File: schemas/context/player-state.schema.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          "description": "The number of times that player state was set on the stream."
+        },
+        "xdm:playerStateTime": {
+          "title": "Show type",
+          "type": "number",
```
Comment (lrosenthol) [inline schemas/context/player-state.schema.json:30]: Should that be an integer?  What about a min/max?

File: schemas/context/player-state.schema.json (hunk 1 above)
Comment (midomir) [reply schemas/context/player-state.schema.json]: Done

File: schemas/context/player-state.schema.json (hunk 2 above)
Comment (midomir) [reply schemas/context/player-state.schema.json]: Done

**Pull request merged**